chart/permissions-api/templates/deployment-worker.yaml

```diff
@@ -58,26 +58,38 @@ spec:
             - name: PERMISSIONSAPI_SERVER_TRUSTED_PROXIES
               value: "{{ join " " . }}"
           {{- end }}
-            - name: PERMISSIONSAPI_EVENTS_SUBSCRIBER_URL
-              value: "{{ .Values.config.events.url }}"
-            - name: PERMISSIONSAPI_EVENTS_SUBSCRIBER_TIMEOUT
-              value: "{{ .Values.config.events.timeout }}"
-            - name: PERMISSIONSAPI_EVENTS_SUBSCRIBER_PREFIX
-              value: "{{ .Values.config.events.prefix }}"
-            - name: PERMISSIONSAPI_EVENTS_SUBSCRIBER_QUEUEGROUP
-              value: "{{ .Values.config.events.queueGroup }}"
+            - name: PERMISSIONSAPI_EVENTS_NATS_URL
+              value: "{{ .Values.config.events.nats.url }}"
+            - name: PERMISSIONSAPI_EVENTS_NATS_SUBSCRIBEPREFIX
+              value: "{{ .Values.config.events.nats.prefix }}"
+            - name: PERMISSIONSAPI_EVENTS_NATS_QUEUEGROUP
+              value: "{{ .Values.config.events.nats.queueGroup }}"
+            - name: PERMISSIONSAPI_EVENTS_NATS_SOURCE
+              value: "{{ .Values.config.events.nats.source }}"
+            - name: PERMISSIONSAPI_EVENTS_NATS_CONNECTTIMEOUT
+              value: "{{ .Values.config.events.nats.connectTimeout }}"
+            - name: PERMISSIONSAPI_EVENTS_NATS_SHUTDOWNTIMEOUT
+              value: "{{ .Values.config.events.nats.shutdownTimeout }}"
+            - name: PERMISSIONSAPI_EVENTS_NATS_SUBSCRIBERFETCHBATCHSIZE
+              value: "{{ .Values.config.events.nats.subscriberFetchBatchSize }}"
+            - name: PERMISSIONSAPI_EVENTS_NATS_SUBSCRIBERFETCHTIMEOUT
+              value: "{{ .Values.config.events.nats.subscriberFetchTimeout }}"
+            - name: PERMISSIONSAPI_EVENTS_NATS_SUBSCRIBERFETCHDELAY
+              value: "{{ .Values.config.events.nats.subscriberFetchDelay }}"
+            - name: PERMISSIONSAPI_EVENTS_NATS_SUBSCRIBERACKEXPLICIT
+              value: "{{ .Values.config.events.nats.subscriberAckExplicit }}"
+            - name: PERMISSIONSAPI_EVENTS_NATS_SUBSCRIBERMANUALACK
+              value: "{{ .Values.config.events.nats.subscriberManualAck }}"
```

Comment:
Looks like these should be updated to match the current config items in the events package.